Automattic/woocommerce-payments
# Add icons as a new column

Fixes https://github.com/Automattic/woocommerce-payments/issues/1989

#### Changes proposed in this Pull Request
- Add WCPay logos to the right of the "WooCommerce Payments" text in `wp-admin/admin.php?page=wc-settings&tab=checkout&section`. 
- Replace "Credit card" to "Popular payment methods" per design.

#### Testing instructions
1. Go to `wp-admin/admin.php?page=wc-settings&tab=checkout&section`
2. Make sure the logo shows up on the right as follow:
![image](https://user-images.githubusercontent.com/572862/121249942-6615ff80-c862-11eb-87f7-b946b88a8e8b.png)


-------------------
